tests/test_settlements.py

```diff
@@ -0,0 +1,190 @@
+import json
+
+import pytest
+
+from mollie.api.error import IdentifierError
+from mollie.api.objects.settlement import Settlement
+from mollie.api.resources.settlements import Settlements
+from tests.utils import assert_list_object
+
+SETTLEMENT_ID = 'stl_jDk30akdN'
+
+
+def test_settlements_resource_class(client, response):
+    response.get('https://api.mollie.com/v2/settlements', 'settlements_list')
+    client.settlements.list()
+
+    assert isinstance(Settlement.get_resource_class(client), Settlements)
```

Comment:
Normally, the `get_resource_class()` method is used when the list response is paginated, and you call `get_next()` or `get_previous()`. If your goal for this test is coverage for the `get_resource_class` method, that's the way to go.

Reply:
removed it completely